mobile-app/app/screens/AppNavigator/screens/Balances/BalancesNavigator.tsx

```diff
@@ -168,7 +168,7 @@ export function BalancesNavigator (): JSX.Element {
         options={({ route }: { route: any }) => ({
           headerTitle: () => {
             const token = route?.params?.token
-            const Icon = getNativeIcon(token.avatarSymbol)
+            const Icon = getNativeIcon(token.displaySymbol)
```

Comment:
How would this affect DFI (UTXO) and DFI (Token)?

Reply:
we don't have `TokenDetail` for DFI (UTXO) and DFI (Token) so it won't affect there